src/main/resources/lang/strings_en.properties

```diff
@@ -475,3 +475,5 @@ colors.fonts.big=Big
 colors.fonts.mid=Medium
 colors.fonts.small=Small
 colors.fonts.tiny=Tiny
+api.errors.unsupported_java=The selected API requires a 64-bit java, but you ran the bot on a 32-bit one<br/>To get a 64-bit java, consider installing our runtime:
+api.errors.non_ansi_path=The bot folder path contains non-ANSI characters. Rename the folders<br/> where they occur or move the bot folder to D:/ and then run again
```

Comment:
Make these messages the FULL html, with parameters for the things you may want to add separately, you can add parameters like `{0}` or `{1}` etc, then when doing `I18n.get` you add the parameters:

`I18n.get("some.string.resource", "param0", "param1")`